.nf-core.yml

```diff
@@ -1 +1,6 @@
 repository_type: pipeline
+lint:
+  files_unchanged:
+    - .github/workflows/branch.yml
+    - .github/workflows/linting_comment.yml
+    - .github/workflows/linting.yml
```

Comment:
why removing these files from the linting?

Reply:
This was done at the very beginning, but I don't remember the reason. Probably to temporarily overcome a linting issue and we forgot to undo. I will delete these lines